virt/virtual_machines/virt-create-vms.adoc

```diff
@@ -3,70 +3,53 @@
 = Creating virtual machines
 include::_attributes/common-attributes.adoc[]
 :context: virt-create-vms
+:toclevels: 3
 
 toc::[]
 
-Use one of these procedures to create a virtual machine:
+You create virtual machines (VMs) by using the {product-title} web console or from the command line.
 
-* Using the web console *Developer* perspective *+Add* page
-* Quick Start guided tour
-* Quick create from the *Catalog*
-* Pasting a pre-configured YAML file with the virtual machine wizard
-* Using the CLI
+Do not create virtual machines in the default `openshift-*` namespaces. Instead, create a new namespace or use an existing namespace without the `openshift` prefix.
```

Comment:
I feel like this should be in an [IMPORTANT] block, especially if we aren't going to give additional context about why.